Makefile

```diff
@@ -12,9 +12,18 @@ test: build obj/tests/tests
 	@obj/tests/tests
 	@gcovr . $(GCOVR_OPTS)
 
-build:
+z3/z3/build:
+	@cd z3/z3; python scripts/mk_make.py
```

Comment:
> We could make this cd z3/z3 && python scripts/mk_make.py just in case

Fixed in https://github.com/Componolit/AZ3/pull/11/commits/12e8fb675e2f62822899d209923489115056d1ba.

> Should this be python3?

Both `python` and `python3` work for me. Using `python3` would make it break in virtual environments and on distributions that don't use `python3` for the Python 3 executable.

Reply:
If both works, we should stick to just `python`.